apps/ledger-live-mobile/src/screens/Assets/index.tsx

```diff
@@ -68,62 +70,68 @@ function Assets() {
 
   const renderItem = useCallback(
     ({ item }: { item: Asset }) => (
-      <AssetRow asset={item} navigation={navigation} />
+      <AssetRow
+        asset={item}
+        navigation={navigation}
+        sourceScreenName={ScreenName.Assets}
+      />
     ),
     [navigation],
   );
 
   return (
-    <TabBarSafeAreaView>
```

Comment:
Nothing changed on the component tree, expect the addition of the ReactNavigationPerformanceView wrapper.